### DISCUSSION TITLE
Loading time laravel

### DISCUSSION THREAD

**Discussion by Lucas-Joshua** [blade-ui-kit/blade-ui-kit#129] [Q&A]
Hi guys,

So I've just installed blade-ui-kit, and to use the icon that I like to use I needed to use this command:
`composer require codeat3/blade-google-material-design-icons`

And ofcours to use blade-ui-kit this command:
`composer require blade-ui-kit/blade-ui-kit`

After running these two commands my loading time from laravel turned from a few m/s to around the 35 seconds

How do I fix this?

**Comment by davidjr82** [comment]
Have you tried caching it?

https://github.com/blade-ui-kit/blade-icons#caching

**Comment by jantbox** [comment]
Same for me.

I'm developing a dashboard with a complex menu but sometimes Laravel doesn't load the complete menu. If I remove all the @svg tags it works ok. If I change Blade Icons by direct Fontawesome tags it works ok...

Ah! Yes I've tried caching.

**Reply by jantbox** [reply]
Dropping blade-icons due to this... :(
